x-pack/plugins/data_visualizer/public/application/common/components/stats_table/components/field_data_row/document_stats.tsx

```diff
@@ -40,15 +42,47 @@ export const DocumentStat = ({ config, showIcon, totalCount }: Props) => {
       ? `(${roundToDecimalPlace((valueCount / total) * 100)}%)`
       : null;
 
+  const content = (
+    <EuiText size={'xs'}>
+      {fieldFormats
+        .getDefaultInstance(KBN_FIELD_TYPES.NUMBER, [ES_FIELD_TYPES.INTEGER])
+        .convert(valueCount)}{' '}
+      {docsPercent}
+    </EuiText>
+  );
+
+  const tooltipContent =
+    type === SUPPORTED_FIELD_TYPES.TEXT ? (
+      <FormattedMessage
+        id="xpack.dataVisualizer.sampledPercentageForTextFieldsMsg"
+        defaultMessage="The % of documents for text fields is sampled and calculated from {sampledDocumentsFormatted} sample {sampledDocuments, plural, one {record} other {records}}."
```

Comment:
I think it could be a bit simpler.

```suggestion
        defaultMessage="The % of documents for text fields is calculated from a sample of {sampledDocumentsFormatted} {sampledDocuments, plural, one {record} other {records}}."
```